app/controllers/stripe_controller.rb

```diff
@@ -35,6 +35,11 @@ def handle_issuing_authorization_request(event)
 
     approved = ::StripeAuthorizationService::Webhook::HandleIssuingAuthorizationRequest.new(stripe_event: event).run
 
+    if approved
+      ::User::UpdateCardLockingJob.perform_later
+      ::User::SendCardLockingNotificationJob.perform_later
```

Comment:
Could we get a weird condition where the second job runs before the first? I feel like they should either be combined or `::User::UpdateCardLockingJob` should call `::User::SendCardLockingNotificationJob`

Reply:
Oh I see why you aren't doing that - to avoid sending these when someone uploads a receipt - hmm. Maybe `UpdateCardLockingJob` takes a parameter?

Reply:
My concern with that is that then it gets confusing; it checks their mossing receipt count, potentially locks their card, and sends notifications. I would rather keep these separate just so that the naming is clear.